exchanges/gateio/gateio_wrapper.go

```diff
@@ -695,18 +695,19 @@ func (g *Gateio) UpdateOrderbookWithLimit(ctx context.Context, p currency.Pair,
 		Pair:            p.Upper(),
 		LastUpdateID:    o.ID,
 		LastUpdated:     o.Update.Time(),
+		UpdatePushedAt:  o.Current.Time(),
```

Comment:
🗻 Out of scope, but it seems to me that
```suggestion
		LastPushed:  o.Current.Time(),
```

Reply:
https://github.com/thrasher-corp/gocryptotrader/pull/1917/commits/f5a6df3e746b59be6ca20f535f736ac1f5b59304